manifest.toml

```diff
@@ -48,8 +48,8 @@ ram.runtime = "100M"
 
 [resources]
         [resources.sources.main]
-        url = "https://github.com/pixelfed/pixelfed/archive/refs/tags/v0.12.6.tar.gz"
-        sha256 = "4d13a104d3b03d6a8a03db2f9114b56fcaca69d52e64334a400ce98c85cbf3ac"
+        url = "https://github.com/pixelfed/pixelfed/archive/5efc09beea56e71b01293fc6e8be8ec9fe8e5875.tar.gz"
+        sha256 = "aecc487a4c369e81adf3dd7b0375e26dd86d274b51357ae8bec12d94895254f0"
```

Comment:
I don't like the idea of not sticking to released version, especially because of the risk of unstability/bugs.

However in that case it comes with a serious bug fix.

I'd prefer to have it available in a separate branch, split from this patch.
However I'm not fully against merging it as it is.